### PR TITLE
#3258 react-native bump

### LINKED ISSUE #3258: Dependencies upgrade
## The suggestion

Need to upgrade dependencies. I suspect it might be a good idea to separate this into multiple issues, if there are real code changes required when upgrading react-native.

**Why?**
Cause we're very behind.

## Describe alternatives you've considered
N/A

## Additional context
N/A

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3258 

## Change summary

Bumps RN

## Testing

- [ ] App still worky

### Related areas to think about

Technically flipper works now, but it doesn't work well with realm, still - for me, anyway!

### CODE REVIEW

File: src/widgets/InfoBadge.js [hunk 1]
```diff
@@ -54,7 +58,7 @@ const InfoBadgeComponent = ({
           <Badge ref={ref} value={unfinalisedCountText} onPress={show} />
           <Popover
             isVisible={visible}
-            fromView={ref.current}
+            from={ref}
```
Comment (joshxg) [inline src/widgets/InfoBadge.js:57]: Actually this was from a couple prs ago when I pumped `popover-view` - they changed the API

File: src/widgets/KeyboardSpacing.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ export const KeyboardSpacing = React.memo(() => {
     toValue: height,
     duration: 100,
     easing: Easing.linear,
+    useNativeDriver: false,
```
Comment (joshxg) [inline src/widgets/KeyboardSpacing.js:13]: Now required to explicitly state true/false

**Review by joshxg — COMMENTED** [review]
Just followed https://react-native-community.github.io/upgrade-helper/

**Review by mark-prins — APPROVED** [review]
LGTM!

**Pull request merged**